### PR TITLE
Participant reset

### PR DESCRIPTION
**Pull request by amber-emmes**
Adds dev-only button to reset a participant back to a just-verified status using new connectFaaS endpoint. (Part of [1084](https://github.com/episphere/connect/issues/1084).)

### CODE REVIEW

File: src/participantDetails.js [hunk 1]
```diff
@@ -107,6 +109,30 @@ export const render = (participant, changedOption) => {
     return template;
 }
 
+const resetParticipantConfirm = (originalHTML) => {
```
Comment (anthonypetersen) [inline src/participantDetails.js:112]: `originalHTML` doesn't appear ot be used?

File: src/participantDetails.js [hunk 2]
```diff
@@ -282,6 +308,24 @@ const renderBackToSearchDivAndButton = () => {
     `;
 };
 
+const renderResetUserButton = (participantUid) => {
+    if(location.hostname === 'localhost' || location.hostname === '127.0.0.1' || location.host === urls.stage) {
```
Comment (anthonypetersen) [inline src/participantDetails.js:312]: in API PR the end-point looked like it was only going to be enabled for dev?

File: src/participantDetailsHelpers.js [hunk 3]
```diff
@@ -1387,6 +1441,10 @@ const processUserDataUpdate = async (changedUserDataForProfile, changedUserDataF
         });
         return true;
 };
+
+const processUserDataReset = async() => {
```
Comment (anthonypetersen) [inline src/participantDetailsHelpers.js:1445]: needed?

File: src/participantDetails.js [hunk 4]
```diff
@@ -38,6 +38,7 @@ export const renderParticipantDetails = (participant, changedOption) => {
     hideUneditableButtons(participant, changedOption);
     localStorage.setItem("participant", JSON.stringify(participant));
     changeParticipantDetail(participant,  changedOption, originalHTML);
+    resetParticipantConfirm(originalHTML);
```
[the remark this answers is not in the document]
Comment (amber-emmes) [reply src/participantDetails.js]: Removed.

File: src/participantDetailsHelpers.js (hunk 3 above)
Comment (amber-emmes) [reply src/participantDetailsHelpers.js]: Oops, nope. Removed.

**Review by anthonypetersen — APPROVED** [review]

**Review by bransteitterbr — APPROVED** [review]

**Pull request merged**